pkg/syncer/volume_health_reconciler.go

```diff
@@ -41,6 +42,54 @@ type VolumeHealthReconciler interface {
 	Run(ctx context.Context, workers int)
 }
 
+// Map of volume handles to the list of PV's that point to this volume handle.
+// Keys are strings representing volume handles.
+// Values are list of strings representing PV names that reference this volume.
+type volumeHandleToPVs struct {
+	*sync.RWMutex
+	items map[string][]string
```

Comment:
Can we use `sync.Map` to avoid wrapping map into volumeHandleToPVs with `*sync.RWMutex`?

Reply:
If you look at the `sync.Map` [documentation](https://golang.org/pkg/sync/#Map), it says
```
...
The Map type is specialized. Most code should use a plain Go map instead, with separate locking or coordination, for better type safety and to make it easier to maintain other invariants along with the map content.
The Map type is optimized for two common use cases: (1) when the entry for a given key is only ever written once but read many times, as in caches that only grow 
...
```
For our use case, we can write multiple times to a given key (multiple PVs can refer the same volumeHandle). 
Also, `sync.Map` does not implement any `LoadAndStore` type of method that would atomically add an entry to the map.